locale/ar/docs/guides/debugging-getting-started.md

```diff
@@ -214,5 +214,5 @@ ssh -L 9221:localhost:9229 user@remote.example.com
 
 <!-- refs -->
 
-[بروتوكول المدقق]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
+[Inspector Protocol]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
```

Comment:
Please change this [line](https://github.com/nodejs/nodejs.org/pull/3152/files#diff-346297c51ece9a9262c1aefa70e3ac18R213)

Reply:
Not sure I follow, this is a link used Line 61

Reply:
Oops sorry! 🤦

Reply:
No worries, could have also changed the anchor on 61, but it doesn't look like there is a standard pattern for the translations right now